config/site_settings.yml

```diff
@@ -2288,6 +2288,9 @@ search:
     default: 1.2
     hidden: true
     validator: "CategorySearchPriorityWeightsValidator"
+  search_page_size:
+    default: 50
+    hidden: true
```

Comment:
Good idea to keep this hidden, generally any of these pagination/rate limiting type ones should be hidden since they can heavily impact performance if changed to a bad value